.eleventy.js

```diff
@@ -83,9 +84,16 @@ module.exports = function(config) {
       });
     },
   };
+  const markdownItAttrsOpts = {
+    leftDelimiter: '{:',
+    rightDelimiter: '}',
+    allowedAttributes: [],
```

Comment:
@robdodson - I've left this blank, so folks can technically apply other attributes to elements, for example they could add a class, or `data-`, etc. If you want to lock it down to just `id`, we can easily do that, but the flexibility is nice. I didn't ever see it abused on WebFu, so I think it should be pretty safe.

Reply:
I updated the comment so it's scoped to `id`, `class`, and `data-*`. I figure it's nice to lock it down because if we start doing localization it might be easy for someone to sneak in an `onload` or something if we're not paying close attention.